lab02-kaylee/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

This is a top level comment that you should have code in your index.js where you imported your functions from the lib folder. 

Per instructions:
```
make a start script for running your index.js in node
```
Understandably, this isn't that clear, but it does imply that there should be functionality in your ``index.js`` module so that when I type ``node index.js`` in my terminal, I should see some output, i.e. ``console.logs``. (This is one of the exceptions that ``console.logs`` are OK since in order to just see some output on the screen). :)